content/docs/csidriver/installation/operator/_index.md

```diff
@@ -266,25 +266,69 @@ The CSI Drivers installed by the Dell CSI Operator can be updated like any Kuber
 
 **NOTES:** 
 1. If you are trying to upgrade the CSI driver from an older version, make sure to modify the _configVersion_ field if required.
-2. The parameter "dnsPolicy: ClusterFirstWithHostNet" should be added as follows:
-```
-driver:
-    configVersion: v5
-    replicas: 2
-    dnsPolicy: ClusterFirstWithHostNet
-    common:
-```
+   ```yaml
+      driver:
+        configVersion: v2.1.0
+   ```
+2. Volume Health Monitoring feature is optional and by default this feature is disabled for drivers when installed via operator.
+   To enable this feature, we will have to modify the below block while upgrading the driver.To get the volume health state add 
+   external-health-monitor sidecar in the sidecar section and `value`under controller set to true and the `value` under node set 
+   to true as shown below:
+    i. add controller and node section as bellow:
+    ```yaml
+        controller:
+          envs:
+            - name: X_CSI_ENABLE_VOL_HEALTH_MONITOR
+              value: "true"
+        dnsPolicy: ClusterFirstWithHostNet
+        node:
+          envs:
+            - name: X_CSI_ENABLE_VOL_HEALTH_MONITOR
+              value: "true"
+    ```
+   ii. update the sidecar versions and Add external-health-monitor sidecar to enable health monitor of CSI volumes from Controller plugin:
```

Comment:
update the sidecar versions and Add external-health-monitor sidecar to enable health monitor of CSI volumes from Controller plugin: -> Update the sidecar versions and add external-health-monitor sidecar if you want to enable health monitor of CSI volumes from Controller plugin:

Reply:
tc